docs/source/devguide/contributing.rst

```diff
@@ -0,0 +1,114 @@
+.. _devguide_contributing:
+
+======================
+Contributing to OpenMC
+======================
+
+Thank you for considering contributing to OpenMC! We look forward to welcoming
+new members to the community and will do our best to help you get up to speed.
+The purpose of this section is to document how the project is managed: how
+contributions (bug fixes, enhancements, new features) are made, how they are
+evaluated, who is permitted to merge pull requests, and what happens in the
+event of disagreements. Once you have read through this section, the
+:ref:`devguide_workflow` section outlines the actual mechanics of making a
+contribution (forking, submitting a pull request, etc.).
+
+The goal of our governance model is to:
+
+- Encourage new contributions.
+- Encourage contributors to remain involved.
+- Avoid unnecessary processes and bureaucracy whenever possible.
+- Create a transparent decision making process which makes it clear how
+  contributors can be involved in decision making.
+
+Overview
+--------
+
+OpenMC uses a liberal contribution model for project governance. Anyone involved
+in development in a non-trivial capacity is given an opportunity to influence
+the direction of the project. Project decisions are made through a
+consensus-seeking process rather than by voting.
+
+Terminology
+-----------
+
+- A *Contributor* is any individual creating or commenting on an issue or pull
+  request.
+- A *Committer* is a subset of contributors who are authorized to review and
+  merge pull requests.
+- The *TC* (Technical Committee) is a group of committers who have the authority
+  to make decisions on behalf of the project team in order to resolve disputes.
+- The *Project Lead* is a single individual who has the authority to make a final
+  decision when the TC is unable to reach consensus.
+
+Contribution Process
+--------------------
+
+Any change to the OpenMC repository must be made through a pull request (PR).
+This applies to all changes to documentation, code, binary files, etc. Even long
+term committers and TC members must use pull requests.
+
+No pull request may be merged without being reviewed.
```

Comment:
I don't see a rule saying "no PR merged by its creator / review by the PR creator doesn't count as the required one". I'm not sure whether this was a rule before or not

Reply:
I've added the word "independently" here.